src/EnergyPlus/StandardRatings.cc

```diff
@@ -531,13 +531,14 @@ namespace StandardRatings {
                 Par(2) = EvapOutletTemp;
                 Par(3) = Cp;
                 Par(4) = ReducedPLR[RedCapNum];
-                Par(5) = EvapVolFlowRate * Rho;
+                Par(5) = CondVolFlowRate * Rho;
                 Par(6) = CapFTempCurveIndex;
                 Par(7) = EIRFTempCurveIndex;
                 Par(8) = EIRFPLRCurveIndex;
                 Par(9) = RefCap;
                 Par(10) = RefCOP;
                 Par(11) = OpenMotorEff;
+                Par(12) = ChillerCapFT_rated;
                 CondenserOutletTemp0 = EnteringWaterTempReduced + 0.1;
                 CondenserOutletTemp1 = EnteringWaterTempReduced + 10.0;
                 General::SolveRoot(state,
```

Comment:
CalcChillerIPLV is only called for ChillerElectricEIR and ChillerReformulatedEIR. So should probably change line 584 below to assert(false) so future developers know to fix something.

Reply:
Sounds good! I just pushed that change.